test/quantization/fx/test_quantize_fx.py

```diff
@@ -1821,21 +1845,180 @@ def forward(self, x):
                 # should not crash as in https://github.com/pytorch/pytorch/issues/75825
                 prepare_fx(m, qconfig_dict)
 
-    def test_qconfig_dict_validity(self):
-        r"""
-        Verifies that if a user passes an invalid key or makes a typo when
-        constructing a qconfig_dict, an error will be thrown and users will be
-        notified of what keys are supported.
+    def test_qconfig_mapping_set_global(self):
+        qconfig = QConfig(123, "global")
```

Comment:
this may work now because QConfig does not typecheck, but at some point this may break.  It might make sense to just use a real constructor.